ajax/libs/WhitestormJS/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "WhitestormJS",
+  "filename": "whitestorm.js",
+  "version": "1.2.6",
+  "homepage": "https://whsjs.readme.io/docs",
+  "author": "sasha240100 <siteprogcom@gmail.com>",
```

Comment:
@extend1994 Use info from `package.json`.
So, yes, correct one is `whs`
I want to be able to use latest whs from beta branch with CDN, can I?